components/filesystem.rst

```diff
@@ -103,6 +103,10 @@ by the third boolean argument::
     // image.jpg will be overridden
     $fileSystem->copy('image-ICC.jpg', 'image.jpg', true);
 
+.. tip::
+
+    Take care remove "../" from path with PHP function "realpath" before set parameters because in some case writing get exception.
```

Comment:
`Take care to remove "../" from the path when using the php function `realpath`, you should not use a relative path in this case, because when writing into the filesystem it will trigger an exception.`